docs/src/manifolds/symmetricpositivedefinite.md

```diff
@@ -21,15 +21,15 @@ Private=false
 Filter = t -> t !== mean
 ```
 
-## Default metric: the linear affine metric
+## Default metric: the affine invariant metric
 
 ```@autodocs
 Modules = [Manifolds]
 Pages = ["manifolds/SymmetricPositiveDefiniteLinearAffine.jl"]
```

Comment:
```suggestion
Pages = ["manifolds/SymmetricPositiveDefiniteAffineInvariant.jl"]
```
I haven't tested but that looks right.

Reply:
again, wellSpotten, I had hoped having found all with my search and replaced, but might have missed this.

Reply:
I just checked if the docs get rendered correctly and they weren't, so I looked what might be the reason :slightly_smiling_face: .

Reply:
...and there was one more reference to update :smile: